Article/Article.js

```diff
@@ -1,20 +1,33 @@
 // Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(article) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.article = article;
```

Comment:
great job making this more descriptive, little things like this can help you visualize what it is you're focusing your attention on.